lstchain/calib/camera/time_correction_calculate.py

```diff
@@ -48,15 +49,10 @@ class TimeCorrectionCalculate(Component):
                        help='Number of capacitors (1024 or 4096). Default 1024.'
                        ).tag(config=True)
 
-    window_width = Int(default_value=7,
-                       help='Define the width of the integration window'
-                      ).tag(config=True)
-
-    window_shift = Int(
-                      default_value=3,
-                      help='Define the shift of the integration window'
-                      'from the peak_index (peak_index - shift)'
-                      ).tag(config=True)
+    charge_product = Unicode(
+        'LocalPeakWindowSum',
```

Comment:
Shall this be hardcoded?

Reply:
This extractor is not suppose to change often, but why do you want to have it hardcoded?